client/packages/purchase_orders/src/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { LocaleKey, useTranslation } from '@common/intl';
+import { PurchaseOrderNodeStatus } from '@common/types';
+
+const statusTranslation: Record<PurchaseOrderNodeStatus, LocaleKey> = {
+  NEW: 'label.new',
+  AUTHORISED: 'label.authorised',
+  CONFIRMED: 'label.confirmed',
+  FINALISED: 'label.finalised',
+};
+
+export const getStatusTranslator =
```

Comment:
Since this is a new "Status" type, needs a new translator.

Basically copied from Invoice though